week2/class.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+        <script>
+    var names = ["Tim", "Timmy", "Timy", "Timm", "Timantha", "Timbo", "Timmey", "Timmie", "Timey", "Timie", "Timmbo", "Timi", "Timmi", "Timei", "Timmei"];
+    var students = 15;
```

Comment:
why not use `names.length`? That's much better than harcoding `15`, and it will always be the right value if you add or remove elements from `names`